homeassistant/components/media_player/bluesound.py

```diff
@@ -667,15 +808,99 @@ def supported_features(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        sleep = 0
+        if self._status is not None:
+            sleep = self._status.get('sleep', 0)
+
         return {
             ATTR_MODEL: self._model,
             ATTR_MODEL_NAME: self._model_name,
             ATTR_BRAND: self._brand,
+            ATTR_SLEEP: sleep
```

Comment:
What is sleep?

Reply:
If this is a value that is automatically reduced / increased based on time, it's not allowed in the state attributes.

Reply:
Sleep is a sleep timer. The value is automatically reduced, where am I suppose to place this kind of information? What's the difference from for example 'media_position' ? 'media_position' too is automatically increased.

Reply:
FYI - I haven't changed this yet, even though github seems to think I changed this.

Reply:
media position should not be automatically updated on each update. We store a time once and store a progress_updated_at. The frontend calculates the right value.

Reply:
Let's remove sleep from the attributes.